### PR TITLE
Fix ROOT6 -> ROOT5 auto merge

### PR DESCRIPTION
**Pull request by wddgit**
Fix auto merge problems of PR 8559 for the merge
from 7_5_X to 7_5_ROOT5_X. These problems were
expected  and are related to necessary differences
in data format checksums and versions.

I recommend these be immediately merged as the current
ROOT5 build is broken and this is code we knew
in advance would not merge.

### CODE REVIEW

**Comment by Dr15Jones** [comment]
@cmsbuild is the bot running?

File: DataFormats/Common/interface/RefVector.h [hunk 1]
```diff
@@ -157,7 +157,7 @@ namespace edm {
                   FillViewHelperVector& helpers) const;
 
     //Needed for ROOT storage
-    CMS_CLASS_VERSION(10)
+    CMS_CLASS_VERSION(12)
```
Comment (Dr15Jones) [inline DataFormats/Common/interface/RefVector.h]: Are the ROOT6 version numbers larger? That is what we were hoping to do.

**Comment by wddgit** [comment]
Yes. The ROOT 6 version number for both RefVector and RefVectorBase was 11 and is now 13.

**Comment by Dr15Jones** [comment]
please test

**Comment by Dr15Jones** [comment]
+1

**Comment by wddgit** [comment]
I think the unit test failure is unrelated to this PR and we should go ahead and merge this anyway to get things to compile. I will look at the unit test failure and try to debug it tomorrow.

**Pull request merged**